templates/bake/element/form.twig

```diff
@@ -23,6 +24,7 @@
                 {{- "\n" }}
             {%- endif %}
         {%- endif %}
+        {% endif %}
```

Comment:
This should probably be `{%- endif %}` to keep the whitespacing as before.

Reply:
Done thanks! I'm pretty new to twig